### ISSUE TITLE
PhoneGap events not firing

### ISSUE THREAD

**Issue by YodasWs** [YodasWs/incremental-game#40]
It appears that the PhoneGap events aren't firing properly. My game doesn't respond to `pause`, `backbutton`, or `menubutton` events.

**Comment by YodasWs** [comment]
This relates to Issues #22 and #23.

**Comment by YodasWs** [comment]
I've just realised that PhoneGap Build is probably placing their phonegap.js in the root directory rather than in the `www` directory... I wish PhoneGap Build would let me declare which files to add/ignore and what the starting page file is, but until then...

**Comment by YodasWs** [comment]
Problem found: PhoneGap Build does not appear to be including phonegap.js and their docs are outdated...

**Comment by YodasWs** [comment]
Okay, so I was having problems not loading `phonegap.js` and I've restructured the files a bit.

PhoneGap Build moves the `www/css` and `www/js` folders along with `www/index.html` down to the main app folder but keeps the `www/ing` folder in `www/img`... It also copies everything from the project root directory. I'm glad I added the `.pgbomit` files to the other directories. I need to see if I can get PhoneGap Build to ignore those files, too.

But my restructuring means that it does handle the `deviceready` event. However, the `device` object is not loaded when `document.readyState` is complete. Some further investigation here is necessary because I use `device.platform` to load platform-specific CSS and JavaScript.
